src/qcodes/instrument_drivers/cryomagnetics/Cryomagnetics4G_visa.py

```diff
@@ -0,0 +1,287 @@
+import re
+import time
+from dataclasses import dataclass
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Enum, Numbers
```

Comment:
utils.validators is a soft deprecated alias. Please import directly from qcodes.validators